mrp_bom_hierarchy/model/mrp_bom.py

```diff
@@ -164,105 +164,3 @@ def action_open_product_other_bom_tree_view(self):
             "[('id', 'in', [" + ",".join(map(str, product_bom_ids.ids)) + "])]"
         )
         return res
-
-
-class MrpBomLine(models.Model):
-    _inherit = "mrp.bom.line"
-
-    has_bom = fields.Boolean(
-        string="Has sub BoM",
-        compute="_compute_child_bom_id",
-    )
-
-    @api.depends("product_id", "bom_id")
-    def _compute_child_bom_id(self):
-        res = super()._compute_child_bom_id()
-        for line in self:
-            line.has_bom = bool(line.child_bom_id)
-        return res
-
-    def action_open_product_bom_tree_view(self):
-        self.ensure_one()
-        res = self.env["ir.actions.actions"]._for_xml_id("mrp.mrp_bom_form_action")
-        if len(self.child_bom_id) == 1:
```

Comment:
@quentinDupont 

Was it on purpose to remove this check ?

Reply:
damn i don't remember sorry

Reply:
I have restored it in v18: https://github.com/OCA/manufacture/pull/1551